doc/architecture/verifiable_event_store/README.md

```diff
@@ -1,37 +1,33 @@
 # Verifiable Event Store
 
-The verifiable event store container has the responsibility to store events as public date.
+The verifiable event store is a container responsible for storing events as public data.
 
-It arranges them in a series of merkle-trees, where each root is published in series to a blockchain,
-this ensures it is tamper proof after the root has been published.
+The event store arranges events in a series of merkle-trees, where each root is published to a blockchain. 
+Doing so ensures that the data is tamper-proof after the root has been published.
 
-The container also has the responsibility to calculate MerkleProofs as requested by users,
-so they can verify their data is part of the tree.
+The container also has the responsibility to calculate _MerkleProofs_ requested by users,
+with the purpose of verifying that their data is part of the tree.
 
-In future is should be posible to get all data stored, so one can follow all changes,
-and verify no illegal operations is done.
+In the future it should be possible to get all data stored, so one can follow all changes,
+and verify that no illegal operations are done.
 
 ![C4 component diagram of the verifiable event store](./component.drawio.svg)
 
 ## Component
 
 ### Batcher
```

Comment:
It seems that in the rest of the documentation, it is described that a set time or number of events can be defined here, but only the batches and their size are described here. Am I mixing something up, or should something be added here?

Reply:
I get your point - it is because we are no longer in the time domain. We are in the validation of transactions domain where we use a merkleproof that consist of batches of transactions in an eventstore, hence batches of events. Maybe we should have a session where we make this distinction concise.